aws_quota/check/__init__.py

```diff
@@ -10,6 +10,7 @@
 from .elasticbeanstalk import *
 from .elb import *
 from .iam import *
+from .lambdas import *
```

Comment:
I originally named this `lambda` but that's a reserved word and it didn't work. I'm open to suggestions for the name of this file.

Reply:
Fine for me :+1: